### DISCUSSION TITLE
Handle handlebars

### DISCUSSION THREAD

**Discussion by Myzel394** [resend/react-email#572] [Q&A]
Many of us will use handlebars in emails like

```
Hey {{ username }},
nice to see you.

Please click on {{ link }} to continue
```

It would be cool if react-email would allow to show placeholder data at those places.

My ideas would be:
* Add sample data in dev somewhere
* Show inputs on the right side that allows us to enter sample data
* Interactively hover over the handlebars and edit them

I know this is quite ambitious, but just wanted to tell you about this idea :D

**Comment by bukinoshita** [comment]
Does the React props works in this case?

```tsx
<MyEmail username="Bu Kinoshita" link="https://resend.com"  />
```

and the email looks like


```tsx
const MyEmail = ({ username, link }) => (
  <Html>
    <Body>
      <Container>
        <Text>Hi, {username}</Text>
        <Text>Please click on <Link href={link}>link</Link> to continue.</Text>
      </Container>
    </Body>
  </Html>
)
```

**Comment by Myzel394** [comment]
Where do I have to use `<MyEmail />`? 

Note that I'm using FastAPI with Postfix to send messages. Because of tbat I will probably just generate my emails and use the built version of them (because of that I need to use handlebars as I replace them in FastAPI).

**Comment by semoal** [comment]
The main idea to use React for emails (apart of dx) is using his template engine. Using handlebars it’s a shoot in your feet.
You could keep the the handlebars expressions as string in the react dim, render to string and then again with handlebars.

**Comment by alikazai** [comment]
when you export the files its currently exporting the properties to  why not have a flag on the cli to out put as handlebars with the property name inside.
